intl/msg_hash_us.h

```diff
@@ -1448,6 +1452,18 @@ MSG_HASH(
    MENU_ENUM_SUBLABEL_INPUT_DRIVER,
    "Input driver to use. Some video drivers force a different input driver."
    )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_INPUT_DRIVER_UDEV,
+   "The udev input driver uses the recent evdev joypad API for joystick support. It supports hotplugging and force feedback.\nThe driver reads evdev events for keyboard support. It also supports keyboard callback, mice and touchpads.\nBy default in most distros, /dev/input nodes are root-only (mode 600). You can set up a udev rule which makes these accessible to non-root."
+   )
+MSG_HASH(
+   MENU_ENUM_LABEL_HELP_INPUT_DRIVER_LINUXRAW,
+   "The linuxraw input driver requires an active TTY. Keyboard events are read directly from the TTY which makes it simpler, but not as flexible as udev. Mice, etc, are not supported at all.This driver uses the older joystick API (/dev/input/js*)."
```

Comment:
Not a Linux user, so... Can we assume a Linux user knows what a "TTY" is"?
Also, missing space after "supported at all."

Reply:
Well, of course not. But it is one of those things many Linux users should learn. It's just one of those things people will have to look up, I'm afraid.